decisions/0004-product-source-field.rst

```diff
@@ -0,0 +1,28 @@
+4. Adding Product Source to Courses on Course Creation via Publisher
+--------------------------------------------------------------------
+
+Status
+------
+
+Accepted (March 2023)
+
+Context
+-------
+Owing to the platform focused marketing and revenue strategies, now there is a massive opportunity to improve the marketplace architecture. As muliple LoBs plugged into edx marketplace for marketing their products, the marketplace architecture needs to be improved 
+to support the new business requirements and allow them to create their products through publisher. Currently, only the legacy edx products are created through publisher with empty product source field. 
+In order to maintain the distinction between the different LoBs products, we need to add a new field in the course creation form in publisher to allow the course teams to select the product source for their course.
+
+Decision
+--------
+Adding a new field in the course creation form in publisher to allow the course teams to select the product source for their course. The Product Source field is a drop-down list shown on the course creation page that allows you to select the source of the product.
+As the multiple LoBs plugged to the system, and the products are coming from 3rd party different sources other than legacy edx products, this field is used to identify the source of
+the product. The product source field is hidden for the time being and we are adding product source default value as "edx" for the courses created through publisher on the request of the PCs as Publisher is a self-service tool for all edX partners and it might confuse them 
+if they see the product source field on the course creation form as the publisher is currently used to publish internal legacy edx products only.
```

Comment:
"as the publisher is currently used to publish internal legacy edx products only." seems redundant in the same sentence. Can be removed.